core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/AwsV4AuthScheme.java

```diff
@@ -53,15 +53,11 @@ default String schemeId() {
      * Retrieve the {@link AwsCredentialsIdentity} based {@link IdentityProvider} associated with this authentication scheme.
      */
     @Override
-    default IdentityProvider<AwsCredentialsIdentity> identityProvider(IdentityProviderConfiguration providers) {
-        return providers.identityProvider(AwsCredentialsIdentity.class);
-    }
+    IdentityProvider<AwsCredentialsIdentity> identityProvider(IdentityProviderConfiguration providers);
```

Comment:
note (applies to other schemes too), this is redefined in this sub-interface to give a more specific javadoc than the generic one in the parent interface.